coda-prototype/initUI.js

```diff
@@ -532,15 +516,8 @@ function initUI(dataset) {
                         "timestamp": new Date()
                     });
 
-                    // TODO: There is duplication between here and handleParseError.
-                    // TODO: There should therefore be a function for displaying errors.
-                    let failAlert = $("#alert");
-                    failAlert.removeClass("alert-success").addClass("alert-danger");
-                    let errorMessage = document.createTextNode("Something is wrong with the data format. " +
+                    UIUtils.displayAlertAsError("Something is wrong with the data format. " +
```

Comment:
This error message has terrible usability. What is wrong? What should the user clean up? We really need to be able to at least report the exception here. (I know this came from previous code)

Please file a bug to fix this error message - and let's talk about how to get the data.

Reply:
Filed as #87